### PR TITLE
add reverb/chorus settings for native MIDI

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: src/m_misc.c [hunk 1]
```diff
@@ -2262,6 +2264,20 @@ default_t defaults[] = {
   },
 #endif
 
+  {
```
Comment (fabiangreffrath) [inline src/m_misc.c:2267]: This needs `#ifdef _WIN32` guards.

Comment (rfomin) [reply src/m_misc.c:2270]: Fixed in https://github.com/fabiangreffrath/woof/pull/724/commits/6ddc5e75c6c51dcede3e755209a602d6e8b9ba96

File: src/i_winmusic.c [hunk 2]
```diff
@@ -450,11 +453,11 @@ static void I_WIN_StopSong(void *handle)
         midiOutShortMsg((HMIDIOUT)hMidiStream, msg);
 
         // reset reverb to 40 and other effect controllers to 0
-        msg = MIDI_EVENT_CONTROLLER | i | 0x5B << 8 | 0x28 << 16; // reverb
```
[the remark this answers is not in the document]
Comment (rfomin) [reply src/i_winmusic.c:453]: Fixed in https://github.com/fabiangreffrath/woof/pull/724/commits/16e1f4a96c31a3c24faff414636a3e883ca9350a

**Comment by fabiangreffrath** [comment]
Thank you!

**Pull request merged**